docs/reference/monitoring/configuring-filebeat.asciidoc

```diff
@@ -0,0 +1,190 @@
+[role="xpack"]
+[testenv="basic"]
+[[configuring-filebeat]]
+=== Collecting {es} log data with {filebeat}
+
+[subs="attributes"]
+++++
+<titleabbrev>Collecting log data with {filebeat}</titleabbrev>
+++++
+
+You can use {filebeat} to monitor the {es} log files, collect log events, and
+ship them to the monitoring cluster. In 7.2 and later, your recent logs are
+visible on the *Monitoring* page in {kib}.
+
+//NOTE: The tagged regions are re-used in the Stack Overview.
+
+. Verify that {es} is running and that the monitoring cluster is ready to
+receive data from {filebeat}.
++
+--
+TIP: In production environments, we strongly recommend using a separate cluster 
+(referred to as the _monitoring cluster_) to store the data. Using a separate 
+monitoring cluster prevents production cluster outages from impacting your 
+ability to access your monitoring data. It also prevents monitoring activities 
+from impacting the performance of your production cluster. See
+{stack-ov}/monitoring-production.html[Monitoring in a production environment].
+
+--
+
+. Enable the collection of monitoring data on your cluster.
++
+--
+//include::configuring-metricbeat.asciidoc[tag=enable-collection]
+Set `xpack.monitoring.collection.enabled` to `true` on each node in the
+production cluster. By default, it is is disabled (`false`). 
+
+NOTE: You can specify this setting in either the `elasticsearch.yml` on each 
+node or across the cluster as a dynamic cluster setting. If {es} 
+{security-features} are enabled, you must have `monitor` cluster privileges to 
+view the cluster settings and `manage` cluster privileges to change them.
+
+For example, you can use the following APIs to review and change this setting:
+
+[source,js]
+----------------------------------
+GET _cluster/settings
+
+PUT _cluster/settings
+{
+  "persistent": {
+    "xpack.monitoring.collection.enabled": true
+  }
+}
+----------------------------------
+// CONSOLE 
+For more information, see <<monitoring-settings>> and <<cluster-update-settings>>.
+--
+
+. Optional: Collect metrics about {es}. See <<configuring-metricbeat>> or
```

Comment:
I do like the "cross sell" here but I also found it a bit out-of-place as step 3 while setting up logs collection. How about making this a note or tip at the bottom of the page?

Reply:
I had it in my brain somehow that the Monitoring UI wouldn't show up on the monitoring cluster until the .monitoring* indices existed so that's why I tested it that way.  And it seemed to me that the Filebeat output went to filebeat* indices. i.e. I needed to do something to get *other* metrics onto the monitoring cluster before any of these log-related pieces will appear.  I'll retest without sending any other monitoring data to the monitoring cluster to see if I was wrong about all that.

Reply:
You're right. There are steps required to see logs within the Monitoring UI that are separate from these filebeat steps, but maybe we can just link off to some enabling monitoring doc, or leave this as-is, but add a little more explanation of why it's necessary?

Reply:
Thanks for the feedback. I've moved this information to the second-last step.